compiler/src/main/scala/org/scalajs/core/compiler/GenJSCode.scala

```diff
@@ -2802,14 +2802,33 @@ abstract class GenJSCode extends plugins.PluginComponent
          *   {name1: arg1, name2: arg2, ... }
          */
 
+        @tailrec
+        def warnIfDuplicatedKey(pairs: List[(js.StringLiteral, js.Tree)], 
+            namesSeen: Set[String] = Set.empty): Unit = {
+          pairs match {
+            case (js.StringLiteral(keyName), _) :: tail =>
+              if (namesSeen(keyName)) {
+                reporter.warning(pos, 
+                    s"Duplicate key in object literal: $keyName. Only the last occurrence is assigned.")
```

Comment:
I agree.
I'm having trouble getting that position because I get `org.scalajs.core.ir.Position` from the `.pos` on those trees and I require a `scala.reflect.internal.util.Position`. Is there a way to convert them or to get back the other possition?